submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionDocDataRepository.kt

```diff
@@ -54,6 +58,12 @@ class SubmissionDocDataRepository(
     }
 
     fun getProjects(accNo: String): List<DocProject> = submissionRepository.getSubmissionProjects(accNo).projects
+
+    private fun updateSubmissionField(accNo: String, version: Int, field: String, value: Any) {
```

Comment:
no quite sure about this, we save 5% of code that we use but at the same time there is an extra call so method is also 5% more complex. Decisions decisions...

```
        mongoTemplate.updateFirst<DocSubmission>(
            Query(where(SUB_ACC_NO).`is`(accNo).andOperator(where(SUB_VERSION).`is`(version))),
            update(SUB_STATUS, status))
 ```

Reply:
Got it, I'll roll these back